solidity/contracts/Peggy.sol

```diff
@@ -36,6 +37,15 @@ contract Peggy {
 		uint256 _amount,
 		uint256 _eventNonce
 	);
+	event ERC20DeployedEvent(
+		// FYI: Can't index on a string without doing a bunch of weird stuff
```

Comment:
I don't understand this comment, can you elaborate?

Reply:
We have an "indexed" argument on most of our events which makes them easier to look up. for some obscure reasons, strings cannot directly be indexed. You can make it work but i will wait until it's needed